src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
This method shouldn't return anything, but perform the action.
Make return type `void` and print information inside method.